contracts/contracts/strategies/CompoundStrategy.sol

```diff
@@ -194,17 +195,17 @@ contract CompoundStrategy is BaseCompoundStrategy {
     }
 
     /**
-     * @dev Approve the spending of all assets by their corresponding cToken,
+     * @notice Approve the spending of all assets by their corresponding cToken,
      *      if for some reason is it necessary.
      */
     function safeApproveAllTokens() external override {
         uint256 assetCount = assetsMapped.length;
-        for (uint256 i = 0; i < assetCount; i++) {
-            address asset = assetsMapped[i];
-            address cToken = assetToPToken[asset];
+        for (uint256 i = 0; i < assetCount; ++i) {
+            IERC20 asset = IERC20(assetsMapped[i]);
```

Comment:
Just curious, this sort of cast isn't a gas saving probably just nicer to read?

Reply:
casting doesn't cost any gas. It was more consistent with other parts of the code.